.env

```diff
@@ -1,2 +1,2 @@
-SNAPSHOT_NAME=pme
-POLYMESH_VERSION=3.0.0
\ No newline at end of file
+SNAPSHOT_NAME=pmf-3.1
+POLYMESH_VERSION=3.1.0
```

Comment:
this would change it for all the PRs in all of the apps (I don't think we have an override specified anywhere right now), is that intended? :)